portal_tables/sync_datasets.py

```diff
@@ -48,12 +68,17 @@ def add_missing_info(
             except (ValueError, IndexError):
                 pass  # PMID not yet annotated or found in portal table
         datasets.at[_, "pub"] = pub_titles
+        
     return datasets
 
 
 def clean_table(df: pd.DataFrame) -> pd.DataFrame:
     """Clean up the table one final time."""
 
+    df["DatasetGrantNumber"] = df["GrantViewKey"]
+    df["DatasetPubmedId"] = df["PublicationViewKey"]
```

Comment:
nit: instead of creating duplicate columns, you can rename the colnames instead:

```suggestion
    df = df.rename(columns={
        "DatasetGrantNumber": "GrantViewKey",
        "DatasetPubmedId": "PublicationViewKey"
    }]
```

Reply:
Good point, I like that better!